src/components/index/sections/Jumbotron.tsx

```diff
@@ -10,23 +10,27 @@ import Typewriter from '../helpers/Typewriter'
 const useStyles = makeStyles((theme) =>
   createStyles({
     container: {
-      paddingTop: theme.spacing(15),
+      height: theme.spacing(93),
+      padding: theme.spacing(15, 1, 0, 1),
       marginBottom: theme.spacing(12),
       textAlign: 'center',
       backgroundImage: 'url(img/header-image.png)',
       backgroundPosition: 'center',
       backgroundSize: 'cover',
       color: theme.palette.common.white,
-      height: theme.spacing(81),
       [theme.breakpoints.up('md')]: {
-        paddingTop: theme.spacing(35),
         height: theme.spacing(100),
+        paddingTop: theme.spacing(31),
+      },
+      [theme.breakpoints.up(1600)]: {
+        height: theme.spacing(119),
```

Comment:
When we're dealing with sizes larger than `spacing(20)` i think we can go directly to the expected size in `rem` or `px` 

The cognitive understanding of how much `spacing(119)` is might be too hard as we need to multiply 119 * 8 to get the figure 952 pixels. We might shoot directly for `950px`

The spacing fn is more useful when we're dealing with gaps, paddings and margins or square objects.